var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,96 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
```

Comment:
ah dang!
https://github.com/ornladios/ADIOS/issues/84

and pew!
http://software.llnl.gov/spack/packaging_guide.html?highlight=autotools#gnu-autotools
```
The setup functionality is currently only available for CMake-based packages. Extending this functionality to GNU Autotools-based packages would be easy (and should be done by a developer who actively uses Autotools). Packages that use non-standard build systems can gain setup functionality by subclassing StagedPackage directly.
```